webapp/sources/rudder/rudder-web/src/main/elm/healthcheck/sources/DataTypes.elm

```diff
@@ -0,0 +1,32 @@
+module DataTypes exposing (..)
+
+import Http exposing (Error)
+type alias Healthcheck = List Check
```

Comment:
I don't think we need that type, use "List Check" instead.

Maybe if it were named HealthChecks but I think List Check is clearer